modules/cgrindel_bazel_starlib/0.14.3/MODULE.bazel

```diff
@@ -0,0 +1,116 @@
+module(
+    name = "cgrindel_bazel_starlib",
+    version = "0.14.3",
+)
+
+bazel_dep(
+    name = "bazel_skylib_gazelle_plugin",
+    version = "1.4.1",
+    dev_dependency = True,
+)
+
+bazel_dep(
+    name = "rules_go",
+    version = "0.38.1",
+    repo_name = "io_bazel_rules_go",
+)
+bazel_dep(
+    name = "gazelle",
+    version = "0.29.0",
+    repo_name = "bazel_gazelle",
+)
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
+bazel_dep(
+    name = "stardoc",
+    version = "0.5.3",
+    repo_name = "io_bazel_stardoc",
+)
+bazel_dep(
+    name = "buildifier_prebuilt",
+    version = "6.0.0.1",
+)
+bazel_dep(name = "platforms", version = "0.0.6")
+
+go_sdk = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")
+go_sdk.download(name = "go_sdk")
```

Comment:
The next version of rules_go will disallow specifying SDK names from non-root modules. rules_go always provides the `go_default_sdk` repo via this extension which you can use without adding a tag. Sorry for the inconvenience, we are still trying to get the API just right.

Reply:
OK. Tracking the issue in cgrindel/bazel-starlib#248